tock/tock/context_processors.py

```diff
@@ -0,0 +1,13 @@
+def version_url(request):
+    import re
+    from django.conf import settings
+    base_url = 'https://github.com/18F/tock/'
+    name = settings.VERSION
+    response = {
+        'release_url': '%stree/%s' % (base_url, name),
+        'release_name': settings.VERSION,
+    }
+    if re.match('/v20[1-9][0-9][0-9]+\.[0-9]+/', name):
```

Comment:
Rather than regexing the tag pattern, would it be adequate (and clearer) to just say `if name.startswith('v20'):`

Reply:
I'm using the same regular expression in the `.circleci/config.yml` to match versions for tag filters. I wanted to maintain consistency on how the release tags are matched. wdyt? There are surely better regular expressions here, but maybe a comment about how it matches in that other file is appropriate there?

Reply:
I think that's a perfectly good reason for doing it this way. The startwith() might be a bit simpler and avoid my regexitis, but this will be more consistent as well as more reliable.